src/transactions/test/CreateAccountTests.cpp

```diff
@@ -198,4 +198,38 @@ TEST_CASE("create account", "[tx][createaccount]")
             }
         });
     }
+    SECTION("too many sponsoring")
+    {
+        auto a2 = root.create(
```

Comment:
Not necessarily something you need to fix, just something I'm trying to understand. You specifically handle `CREATE_ACCOUNT` on SponsorshipTestUtils.cpp:330 but then you don't actually use that functionality here. Was this just a miss (perhaps you wrote this first) or did it not really fit into the broader functionality of that test (I've experienced that problem)?

Reply:
I originally wanted to use `tooManySponsoring` here, but the create account test was different enough that I decided not to use the helper. After looking at the the helper though, I realized that the create account use case didn't fit in well due to the way I setup the transactions. I used the sponsored account as the tx source account, which doesn't work for create account. Instead I could use the sponsoring account as the tx source account. 

I made this change, along with just using root at the sponsoring account to make things simpler. This allowed me to use `tooManySponsoring` for the create account case.